<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prime</class>
 <widget class="QWidget" name="prime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(18, 78, 102);


</string>
  </property>
  <widget class="QPushButton" name="clean_but">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>500</y>
     <width>351</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font: 30pt &quot;Ubuntu&quot;;
    background-color: transparent;
    border-radius: 20px;
    border: none;
	image: url(:/images/tuneddd.png);
    font-weight: bold;
color:white;
}

QPushButton:focus {
    outline: none;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string>CLEAN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Tune_but">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>351</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font: 30pt &quot;Ubuntu&quot;;
    background-color: transparent;
    border-radius: 20px;
    border: none;
	image: url(:/images/tuneddd.png);
    font-weight: bold;
color:white;
}

QPushButton:focus {
    outline: none;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string>TUNE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prime1_but">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>351</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font: 30pt &quot;Ubuntu&quot;;
    background-color: transparent;
    border-radius: 20px;
    border: none;
	image: url(:/images/tuneddd.png);
    font-weight: bold;
color:white;
}

QPushButton:focus {
    outline: none;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string>PRIME</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>-30</y>
     <width>871</width>
     <height>791</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_6">
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(18, 78, 102);


</string>
    </property>
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QLabel" name="process_lab_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:25pt &quot;Ubuntu&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius:20px;
border:none;font:bold;</string>
     </property>
     <property name="text">
      <string>     PROCESS </string>
     </property>
    </widget>
    <widget class="QPushButton" name="start_prime_but_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>620</y>
       <width>321</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
font:20pt &quot;Ubuntu&quot;;
image: url(:/images/g4210.png);
background-color:transparent;
color: rgb(255, 255, 255);
border-radius:20px;
border:none;font:bold;
}
QPushButton:focus {
    outline: none;
    border: none;
}
</string>
     </property>
     <property name="text">
      <string>START PRIME</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="start_check_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>260</y>
       <width>721</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox::indicator {
	color: rgb(255, 255, 255);
    width: 30px;
    height: 30px;
}
QCheckBox {
    font-size: 30px;
	color: rgb(255, 255, 255);

}
</string>
     </property>
     <property name="text">
      <string>Start Irrigation Valve</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="wait_Check_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>360</y>
       <width>751</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox::indicator {
	color: rgb(255, 255, 255);
    width: 30px;
    height: 30px;
}
QCheckBox {
    font-size: 30px;
	color: rgb(255, 255, 255);

}
</string>
     </property>
     <property name="text">
      <string>Wait For Chamber To Be Filled</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="motor_Check_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>381</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox::indicator {
	color: rgb(255, 255, 255);
    width: 30px;
    height: 30px;
}
QCheckBox {
    font-size: 30px;
	color: rgb(255, 255, 255);

}
</string>
     </property>
     <property name="text">
      <string>Start Motor</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="done_Check_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>470</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox::indicator {
	color: rgb(255, 255, 255);
    width: 30px;
    height: 30px;
}
QCheckBox {
    font-size: 30px;
	color: rgb(255, 255, 255);

}
</string>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>560</y>
       <width>761</width>
       <height>51</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <zorder>progressBar_2</zorder>
    <zorder>process_lab_2</zorder>
    <zorder>start_prime_but_2</zorder>
    <zorder>start_check_2</zorder>
    <zorder>wait_Check_2</zorder>
    <zorder>motor_Check_2</zorder>
    <zorder>done_Check_2</zorder>
   </widget>
   <widget class="QWidget" name="tab_8">
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(18, 78, 102);


</string>
    </property>
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QLabel" name="warning_lab_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:25pt &quot;Ubuntu&quot;;
background-color: rgb(36, 31, 49);
color: rgb(255, 255, 255);
border-radius:20px;
border:none;
font:bold;</string>
     </property>
     <property name="text">
      <string>   BEFORE CLEANING</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="Instruction_lab_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>260</y>
       <width>741</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:20px;border:none;
background-color: transparent;
</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:20px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;1.Disconnect the Irrigation/Aspiration connectors from handpiece&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;2.Connect the irrigation,aspiration connectors together&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;3.Disconnect the saline connection from irrigation&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;4.Disconnect the tubing from irriagtion valve&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;5.Dip the irrigation connector in distilled water bowl.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="begin_clean_but_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>570</y>
       <width>331</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
font:20pt &quot;Ubuntu&quot;;
background-color: transparent;
image: url(:/images/g4210.png);
color: rgb(255, 255, 255);
font:bold;
border-radius:20px;
border:none;
}
QPushButton:focus {
    outline: none;
    border: none;
}
</string>
     </property>
     <property name="text">
      <string>Begin Clean</string>
     </property>
    </widget>
    <widget class="QLabel" name="warning_img_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/warning1.png);
BACKGROUND-COLOR:TRANSPARENT;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>651</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);
font:bold;
background-color:transparent;</string>
     </property>
     <property name="text">
      <string> Instructions To Be Followed:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/prime.png);
background-color:transparent;
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>360</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/wifi.png);
background-color:transparent;
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>570</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/clean.png);
background-color:transparent;
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lab_time">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>20</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: transparent;
border:1px solid white;
border-radius:10px;
color: rgb(255, 255, 255);
font:20pt &quot;Ubuntu&quot;;
font-weight: bold;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_date">
   <property name="geometry">
    <rect>
     <x>744</x>
     <y>20</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: transparent;
border:1px solid white;
border-radius:10px;
color: rgb(255, 255, 255);
font:20pt &quot;Ubuntu&quot;;
font-weight: bold;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>20</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:12px;
image: url(:/images/clock.png);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>0</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:12px;
image: url(:/images/calendor.png);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>19</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: transparent;
border:1px solid white;
border-radius:10px;
color: rgb(255, 255, 255);
font:20pt &quot;Ubuntu&quot;;
font-weight: bold;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: transparent;
border:1px solid white;
border-radius:10px;
color: rgb(255, 255, 255);
font:20pt &quot;Ubuntu&quot;;
font-weight: bold;</string>
   </property>
   <item>
    <property name="text">
     <string>Surgeon 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Surgeon 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Surgeon 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Surgeon 4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Surgeon 5</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lbloverall">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1281</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>16</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>1280</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>lab_date</zorder>
  <zorder>lab_time</zorder>
  <zorder>label_5</zorder>
  <zorder>label_4</zorder>
  <zorder>prime1_but</zorder>
  <zorder>tabWidget</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label</zorder>
  <zorder>Tune_but</zorder>
  <zorder>clean_but</zorder>
  <zorder>lbloverall</zorder>
  <zorder>label_6</zorder>
  <zorder>comboBox_4</zorder>
  <zorder>label_7</zorder>
  <zorder>line</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
